lib/Mojo/WebSocketProxy/CallingEngine.pm

```diff
@@ -144,7 +144,12 @@ sub call_rpc {
 
             my $api_response;
             if (!$res) {
-                warn "WrongResponse [$msg_type]";
+                my $tx = $client->tx;
+                my $details = 'URL: ' . $tx->req->url;
```

Comment:
shouldn't this check if `tx` object exists, like `my $details = $tx ? 'URL: ' . $tx->req->url : '';`